detector/muonSystem/muonSystemMuRWELL_o1_v01.cpp

```diff
@@ -0,0 +1,941 @@
+/*
+ @author: Mahmoud Ali
+mahmoud.ali@cern.ch
+
+Factory for IDEA muon system
+Expected xml structure (the 'sensitive' keyword is optional and defaults to false):
+<detector type="muonSystemMuRWELL_o1_v01" ...>
+  <envelope rmin="..." rmax="..." z="..."  z_offset="..." material="..."/>  <!-- dimensions of the whole detector envelope-->
+  <dimensions x="..." y="..." z="..." z_offset="..." x_offset="..." y_offset="...">  <!--  dimension of the local chamber envelope. x: the thickness of the chamber, y&z: the 2D plane dimensions of the chambers-->
```

Comment:
They seem to be half-lengths? Is it also true for the x-coordinate? I think it worth mentioning this.